apps/docs/content/components/VideoPlayer.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: VideoPlayer
+status: Experimental
+source: https://github.com/primer/brand/blob/main/packages/react/src/VideoPlayer/VideoPlayer.tsx
+storybook: '/brand/storybook/?path=/story/components-videoplayer--playground'
+description: Use the video player component to present videos availible via CDN or local.
+---
+
+import {Label} from '@primer/react'
+
+```js
+import {VideoPlayer} from '@primer/react-brand'
+```
+
+## Examples
```

Comment:
FYI there seems to be a strange hover / overflow related bug in the docs examples 👇 


https://github.com/primer/brand/assets/13340707/cab69536-23fd-48d5-9c61-3b43e9797f33

Reply:
Hiding the overflow seems to clip the tooltip. Is there any other workaround? Do we know why that was occurring on hover to begin with? 

![Screenshot 2023-08-11 at 17 13 36](https://github.com/primer/brand/assets/13340707/12528c3e-a55e-4b2d-93c3-f0edf9838aa5)

Reply:
It seemed to happen because of the tooltip. I think if we want to address this correctly we would need to create a Tooltip component as we have discussed. Then this can be updated.

Reply:
Creating a dedicated tooltip will require considerable effort, so can we offset this particular tooltip from the left by a few pixels in the meantime?